reactivesearch/src/ts/index.ts

```diff
@@ -83,7 +83,7 @@ const context = new Context(window, document);
 
 
 /**/
-const writer = new UbiWriter('http://127.0.0.1:9200', 'ubl_log', queryResolver, sessionResolver,  debug);
+const writer = new UbiWriter('http://127.0.0.1:9200', 'bad_log_shouldnt_exist', queryResolver, sessionResolver,  debug);
```

Comment:
Not sure I undertstand this change?

Reply:
Nothing in that file should be firing yet.  It's from the 1st js demo.  So, if we see that index appear, something's going on that shouldn't be.